docs/hub/datasets-polars-auth.md

```diff
@@ -0,0 +1,26 @@
+# Authentication
```

Comment:
does Polars support authentication from `huggingface-cli login` ? it stores the token at `~/.cache/huggingface/token` where `~/.cache/huggingface` is the default `HF_HOME`.

Reply:
@nameexhaustion do we support this?

Reply:
let's keep things as is for now and we can update later if needed

Reply:
> @nameexhaustion do/can we support this?

made a PR here https://github.com/pola-rs/polars/pull/17881